ansibleflow/run.py

```diff
@@ -18,9 +18,12 @@ def convert_var_filename_to_arg(filename):
     return ' -e @{0}'.format(os.path.abspath(filename))
 
 
-def build_ansible_command(playbook, target, environment):
+def build_ansible_command(playbook, target, environment, arguments):
     command = 'ansible-playbook'
 
+    if arguments.verbosity > 0:
+        command += ' -{0}'.format('v' * arguments.verbosity)
```

Comment:
This is a convenience...it's easier to just apply debug to the executed tasks than to locate the target in a yml file and then re-run the target.